### ISSUE TITLE
Version numbering

### ISSUE THREAD

**Issue by robouden** [Safecast/bGeigieIntegratedSW#1]
Maybe start with V2.0.0? To be not confused with the normal BGeigieNano

**Comment by fakufaku** [comment]
Absolutely. I think we mentioned at some point bumping the major number for hardware changes.
In general we could try to follow something similar to that: http://semver.org/
How about:
1. **Major**: Hardware changes
2. **Minor**: New features, big changes
3. **Patches**: bug fixes and incremental
